Tools/Scripts/libraries/webkitbugspy/webkitbugspy/issue.py

```diff
@@ -278,10 +278,14 @@ def redacted(self):
                 )
 
         match_strings = {self.link: match_string}
-        if self.original:
-            match_strings[self.original.link] = self.original._redaction_match
-        for dupe in self.duplicates or []:
-            match_strings[dupe.link] = dupe._redaction_match
+        for dupe in (self.duplicates or []) + ([self.original] if self.original else []):
```

Comment:
Yikes, that's a gnarly array concat... 

Also, nit: Is `dupe` the right variable name since it contains the original?